presto-main/src/test/java/com/facebook/presto/operator/scalar/AbstractTestFunctions.java

```diff
@@ -94,6 +94,11 @@ protected void assertFunction(String projection, Type expectedType, Object expec
         functionAssertions.assertFunction(projection, expectedType, expected);
     }
 
+    protected void tryEvaluateWithAll(String projection, Type expectedType)
```

Comment:
This is a weird place to put the method (in between other "assert" methods). How about putting it after those?